docs/getting_started/tutorial-hyperbolic-discounting.ipynb

```diff
@@ -0,0 +1,1402 @@
+{
```

Comment:
<p>Use <code>params[["value", "group"]]</code> instead of <code>.loc</code> if you only need columns.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>